### ISSUE TITLE
social buttons are not accessible

### ISSUE THREAD

**Issue by 7mary4** [google/material-design-lite#1058]
The blog template, and possibly others, include social media buttons that have no text node.
`<button class="mdl-mini-footer--social-btn social-btn social-btn__twitter"></button>`
`<button class="mdl-mini-footer--social-btn social-btn social-btn__blogger"></button>`
`<button class="mdl-mini-footer--social-btn social-btn social-btn__gplus"></button>`
You already have class .visuallyhidden in the CSS. This should be fixed by adding a hidden text node
`<button class="mdl-mini-footer--social-btn social-btn social-btn__twitter"><span class="visuallyhidden">Twitter</span></button>`
`<button class="mdl-mini-footer--social-btn social-btn social-btn__blogger"><span class="visuallyhidden">Blogger</span></button>`
`<button class="mdl-mini-footer--social-btn social-btn social-btn__gplus"><span class="visuallyhidden">Google Plus</span></button>`
